src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionWarnOthersViewController.swift

```diff
@@ -155,14 +155,30 @@ private extension ExposureSubmissionWarnOthersViewController {
 							  accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmissionWarnOthers.description),
 						.custom(withIdentifier: CustomCellReuseIdentifiers.roundedCell,
 								configure: { _, cell, _ in
-									guard let cell = cell as? DynamicTableViewRoundedCell else { return }
-									cell.configure(
+									guard let cell0 = cell as? DynamicTableViewRoundedCell else { return }
```

Comment:
I don't quite understand why you started naming the cells `cell0` and `cell1`? 
If you wanted to show that they are different, please use a name that describes them better otherwise just call them both `cell`

Reply:
done. Used explicit names now.